generators/docker-compose/prompts.js

```diff
@@ -287,7 +287,8 @@ function getAppFolders(input, applicationType) {
                     if ((fileData['generator-jhipster'].baseName !== undefined)
                         && ((applicationType === undefined)
                             || (applicationType === fileData['generator-jhipster'].applicationType)
-                            || ((applicationType === 'microservice') && ('gateway' === fileData['generator-jhipster'].applicationType)))) {
+                            || ((applicationType === 'microservice') && ('gateway' === fileData['generator-jhipster'].applicationType))
```

Comment:
it can be 'uaa', too. This actually is why the generator seems not to generate the yaml